docs/keyConcepts.md

```diff
@@ -742,6 +742,25 @@ useful in the following scenarios:
 
 If you need to export raw samples from VictoriaMetrics, then take a look at [export APIs](https://docs.victoriametrics.com/#how-to-export-time-series).
 
+### Query latency
+
+By default, Victoria Metrics does not give back the just-written samples immediately, it returns last results,
+written before duration from `search.latencyOffset` flag (default value - `30s`).
```

Comment:
It seems to me like "duration" term here might be confusing for people not familiar with Go, so it might be better to rewrite this as:
```suggestion
By default, Victoria Metrics does not immediately return the recently written samples. Instead, it retrieves the last results written prior to the time specified by the `search.latencyOffset` flag, which has a default offset of 30 seconds.
```